src/LowLevelMotionController.cpp

```diff
@@ -24,9 +55,46 @@ int main(int argc, char **argv)
 
     AccelerationPlanner<QuadVelocityController> accelerationPlanner(nh, quadController);
 
+    ros::Publisher uav_control_ = nh.advertise<iarc7_msgs::OrientationThrottleStamped>("uav_direction_command", 50);
+
+    // Check for empty uav_control_ as per http://wiki.ros.org/roscpp/Overview/Publishers%20and%20Subscribers
+    // section 1
+    if(!uav_control_)
```

Comment:
I believe this if statement should just be `ROS_ASSERT_MSG(uav_control_, "Could not...");`